### DISCUSSION TITLE
How to set Levels on Docker Logs with GELF

### DISCUSSION THREAD

**Discussion by vatavale** [datalust/seq-tickets#1578] [Help] [answered]
I am trying to set levels of log messages (like Error or Warning) with no luck.

![image](https://user-images.githubusercontent.com/3200219/171475544-7f8c4105-ee92-4f62-8b71-549484c533c8.png)

I am ingesting logs from Docker containers with GELF protocol.

**Comment by KodrAus** [comment, marked as answer]
Hi @vatavale :wave:


Thanks for reaching out! Hmm, are those `level`, `Level`, `_level`, and `@Level` properties all added by you to make Seq pick up the level? If you're in control of those, you can use the name `@l` for the level. For reference, Seq uses [the CLEF format](https://clef-json.org/) for its properties.

We can also detect the presence of a property called `level` or `lvl` and set that to `@l` if your JSON payload doesn't include `@l`. We do this already for `message` to `@m` in the GELF server.

**Reply by vatavale** [reply]
Yes it works! Thank you!
